test/migration/verify/verify_migration_test.go

```diff
@@ -119,6 +122,43 @@ func verifySecondMemberProvisionedSpace(t *testing.T, awaitilities wait.Awaitili
 	cleanup.AddCleanTasks(t, awaitilities.Host().Client, userSignupForSpace)
 }
 
+func verifyProvisionedSubSpace(t *testing.T, awaitilities wait.Awaitilities) {
+	parentSpace, _ := VerifyResourcesProvisionedForSpace(t, awaitilities, migration.ProvisionedParentSpace)
+	userSignupForSpace := checkMURMigratedAndGetSignup(t, awaitilities.Host(), migration.ProvisionedParentSpace)
```

Comment:
unless I've missed something - the parentSpace is created using `createAndWaitForSpace` which doesn't provision a usersignup resource.

Reply:
Yes, it's made with `createAndWaitForSpace`, but the other spaces in migration setup [also call this function][1].

[1]: https://github.com/sadlerap/toolchain-e2e/blob/subspace/test/migration/setup_runner.go#L65-L71

Reply:
My bad, I saw now that it actually creates the MUR and UserSignup [here](https://github.com/codeready-toolchain/toolchain-e2e/blob/e2addcbf280e64bc040a281bf986bc0b273d168f/test/migration/setup_runner.go#L75) 🤦‍♂️ . Sorry for the confusion.

Reply:
`createAndWaitForSpace` calls [`tsspace.CreateMurWithAdminSpaceBindingForSpace`][1], which creates both the `UserSignup` and the `MasterUserRecord` [here][2].

[1]: https://github.com/codeready-toolchain/toolchain-e2e/blob/e2addcbf280e64bc040a281bf986bc0b273d168f/test/migration/setup_runner.go#L75C20-L75C67
[2]: https://github.com/codeready-toolchain/toolchain-e2e/blob/e2addcbf280e64bc040a281bf986bc0b273d168f/testsupport/space/space.go#L180